openedx/core/djangoapps/cors_csrf/admin.py

```diff
@@ -1,7 +1,11 @@
-"""Manage cross-domain configuration. """
+"""
+Manage cross-domain configuration.
+"""
+
 from django.contrib import admin
 from config_models.admin import ConfigurationModelAdmin
-from cors_csrf.models import XDomainProxyConfiguration
+
+from .models import XDomainProxyConfiguration
```

Comment:
My preference is to use relative imports for the Django app itself. It is then clearer what code is external, and what code is part of the app. I also like that it makes it easier to move the code too. I don't have a strong feeling about it though, so I'm willing to be convinced that there are reasons not to do this.